challenges-09.test.js

```diff
@@ -0,0 +1,322 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named countNumberOfElements that, given an array as input, uses reduce to count the number of elements in the array.
+
+Note: You may not use the array's built-in length property.
+------------------------------------------------------------------------------------------------ */
+
+const countNumberOfElements = (arr) => {
+  // Solution code here...
+ return arr.length
+};
```

Comment:
The instructions state that you cannot use array.length. Nawal did the same thing ;)